docs/embeddings.md

```diff
@@ -171,23 +212,27 @@ cohere_ef(texts=["document1","document2"])
 <TabItem value="js" label="JavaScript">
 
 ```javascript
-const {CohereEmbeddingFunction} = require('chromadb');
-const embedder = new CohereEmbeddingFunction("apiKey")
+const { CohereEmbeddingFunction } = require('chromadb');
+const embedder = new CohereEmbeddingFunction('apiKey');
 
-// use directly 
-const embeddings = embedder.generate(["document1","document2"])
+// use directly
+const embeddings = embedder.generate(['document1', 'document2']);
```

Comment:
Can be removed as not used further down the example.

Reply:
Alternatively, add an example of of how embeddings can be inserted into the collection being created.